Calibration/HcalIsolatedTrackReco/src/IPTCorrector.cc

```diff
@@ -78,8 +78,8 @@ void IPTCorrector::produce(edm::StreamID, edm::Event& theEvent, edm::EventSetup
   }
   
   // put the product in the event
-  std::auto_ptr< reco::IsolatedPixelTrackCandidateCollection > outCollection(trackCollection);
-  theEvent.put(outCollection);
+  std::unique_ptr< reco::IsolatedPixelTrackCandidateCollection > outCollection(trackCollection);
+  theEvent.put(std::move(outCollection));
 
```

Comment:
@wmtan
'iptcCollection' is a raw pointer. I think this should be wrapped in a unique_ptr like you do in other places.